javascript/src/HtmlEditorField.js

```diff
@@ -844,7 +844,14 @@ $.entwine('ss', function($) {
 				action = "update";
 			}
 
-			if(href.match(/^mailto:(.*)$/)) {
+            if(href.match(/^mailto:(.*)\?subject=(.*)$/)) {
```

Comment:
I think we can make this more generic to parse the url better and pull out all the query params.